synfig-studio/src/gui/app.cpp

```diff
@@ -2308,34 +2308,49 @@ App::apply_gtk_settings()
-	data += ".button                            { padding-left: 4px; padding-right: 4px; }\n";
-	data += ".button                            { padding-top: 0px; padding-bottom: 0px; }\n";
-	data += ".button *                          { padding-top: 4px; padding-bottom: 4px; }\n";
-	data += ".button > GtkBox                   { padding-top: 0px; padding-bottom: 0px; }\n";
-	data += ".button > GtkBox > *               { padding-top: 4px; padding-bottom: 4px; }\n";
-	data += ".button > GtkLabel                 { padding-top: 0px; padding-bottom: 0px; }\n";
-	data += "GtkComboBox > .button > GtkBox > * { padding-top: 0px; padding-bottom: 0px; }\n";
-	data += ".entry                             { padding-top: 0px; padding-bottom: 0px; }\n";
-	data += "progress, trough 					{ min-height: 20px; }\n";
-#if GTKMM_MAJOR_VERSION < 3 || (GTKMM_MAJOR_VERSION == 3 && GTKMM_MINOR_VERSION < 22)
+//	data += ".button                            { padding-left: 4px; padding-right: 4px; }\n";
+//	data += ".button                            { padding-top: 0px; padding-bottom: 0px; }\n";
+//	data += ".button *                          { padding-top: 4px; padding-bottom: 4px; }\n";
+//	data += ".button > GtkBox                   { padding-top: 0px; padding-bottom: 0px; }\n";
+//	data += ".button > GtkBox > *               { padding-top: 4px; padding-bottom: 4px; }\n";
+//	data += ".button > GtkLabel                 { padding-top: 0px; padding-bottom: 0px; }\n";
+//	data += "GtkComboBox > .button > GtkBox > * { padding-top: 0px; padding-bottom: 0px; }\n";
+//	data += ".entry                             { padding-top: 0px; padding-bottom: 0px; }\n";
+//	data += "progress, trough 					{ min-height: 20px; }\n";
+//#if GTKMM_MAJOR_VERSION < 3 || (GTKMM_MAJOR_VERSION == 3 && GTKMM_MINOR_VERSION < 22)
 	// following css works in old versions of gtk
-	data += "button { padding: 0px; }\n";
-#else
+//	data += "button { padding: 0px; }\n";
+//#else
 	// following css works for gtk 3.22:
-	data += "entry, spinbutton { min-height: 16px; }\n";
-	data += "button { min-height: 16px; min-width: 16px; padding: 0px; }\n";
-#endif
-	data += "button > box { padding: 5px; }\n";
-	data += "button > image { padding: 5px; }\n";
-	data += "combobox > box > button > box { padding-top: 0px; padding-bottom: 0px; }\n";
-	// Fix #810: Insetsetive context menus on OSX
+//	data += "entry, spinbutton { min-height: 16px; }\n";
+//	data += "button { min-height: 16px; min-width: 16px; padding: 0px; }\n";
+//#endif
+//	data += "button > box { padding: 5px; }\n";
+//	data += "button > image { padding: 5px; }\n";
+//	data += "combobox > box > button > box { padding-top: 0px; padding-bottom: 0px; }\n";
+
+	data += "#render-progress * { min-height: 20px; }\n";
+
+	data += ".synfigstudio-efficient-workspace button { padding: 0px; }\n";
+	data += ".synfigstudio-efficient-workspace button > image { padding: 5px; }\n";
+
+	data += "#low-resolution label { padding-left: 5px; padding-right: 5px; }\n";
```

Comment:
Why do we need to set this specific label with a different style?

Reply:
When we set button padding to 0px, its text and icons would touch the edge of the button. Just look at GitHub "Resolve conversation" button below. Try to imagine it's text label without left/right padding. Now that looks bad because the information looks like it was cramped inside.

Anyway, what the previous code did was set the button padding to 0px and its image ie., button icons to 5px. And whoever wrote that code left or forgot to set padding for the text. So I took it up and fixed that :)

Reply:
Couldn't be done by removing the first line of your previous comment?
```
class .synfigstudio-efficient-workspace {
    button        : <value>;   # removing this line?
    button > image: <value>;
}
```

Reply:
If we remove this line we're going to default back to GTK padding which is kinda too big :(
Plus we're unnecessarily adding more padding to button's image.

I'll send you the screenshot...

Reply:
BTW. screenshots were taken with SynfigStudio window maximized

# Default GTK/Adwaita theme without our custom CSS
![01 default gtk](https://user-images.githubusercontent.com/24214509/96673532-5950a400-1384-11eb-942c-7365016d34e5.png)
# `button : <value>` removed
![02 rodolforg](https://user-images.githubusercontent.com/24214509/96673578-6f5e6480-1384-11eb-85b7-59d3c388b30f.png)
# Our current custom CSS applied
![03 css](https://user-images.githubusercontent.com/24214509/96673600-7ab19000-1384-11eb-95fb-4d080c1f6c18.png)

Reply:
Oh, I said it wrong :P 
I meant remove/reduce the padding only for image 'inside' button.
Well, anyway, there should be more issues :P Thank you for clarifying me.

Reply:
Looks nice and clean!

Reply:
> Why do we need to set this specific label with a different style?

Why don't use a generic way like `button > label`?
I'm asking because I don't know, ok? :) Not to complain.

Reply:
Wait I didn't realize I made a mistake here
```css
class .synfigstudio-efficient-workspace {
    button        : <value>;
    button > image: <value>;
}
```
LOL it's
```css
.synfigstudio-efficient-workspace  button  {
    padding: <value>;
```
```css
.synfigstudio-efficient-workspace button > image {
    padding: <value>;
}
```

Sorry, I was a little burned out :P I didn't review my own comments

Hope our conversation doesn't get any more complicated :smile: :rofl: